app/schema.graphql

```diff
@@ -624,6 +624,7 @@ type Span {
 
   """Span attributes as a JSON string"""
   attributes: String!
+  metadata: String
```

Comment:
Needs a description that it's a JSON string

Reply:
addressed in 95666edf